images/slim.sh

```diff
@@ -15,8 +15,14 @@ go install golang.org/dl/go1.19.12@latest && \
 
 # Tooling
 
-curl -sfL https://deb.nodesource.com/setup_18.x  | bash -
-apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -y brotli unzip libuv1-dev libjson-c-dev nettle-dev nodejs
+apt-get update && apt-get install -y ca-certificates curl gnupg
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+
+NODE_MAJOR=20
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list
+
+DEBIAN_FRONTEND="noninteractive" apt-get install -y brotli unzip libuv1-dev libjson-c-dev nettle-dev nodejs npm
 npm install -g npm@9.6.7
 npm install -g pnpm@v8
```

Comment:
Are npm and pnpm also the latest lts at the moment?

Reply:
latest npm is at 10.2.5.
I think pnpm might be at latest since we've specified the latest major and left out minor and patch

Reply:
let's use npm appropriate for the node release in that case